### PR TITLE
More Clippy warnings

### PR DESCRIPTION
**Pull request by jsulmont**

### CODE REVIEW

File: src/parser/ast.rs [hunk 1]
```diff
@@ -95,6 +95,7 @@ pub struct VariableDeclaration {
 }
 
 #[derive(Debug, PartialEq)]
+#[allow(clippy::vec_box)]
```
Comment (seanyoung) [inline src/parser/ast.rs]: Actually I think clippy is right here. This should be fixed (and I think I already fixed this)

**Comment by seanyoung** [comment]
Thanks for the PR, looks great!

Just one small comment.

Hyperledger needs a DCO, please add a Signed-off-by: to your commit. Thanks again!

**Comment by seanyoung** [comment]
Looks great, thanks. Just needs rebasing.

**Comment by jsulmont** [comment]
@seanyoung not sure where you need this rebased

**Pull request merged**